README.md

```diff
@@ -1,2 +1,32 @@
 # ingest-adapter
-An application for pushlishing HyP3 output products for ingest into ASF's data catalog
+
+An application for publishing HyP3 output products for ingest into ASF's data catalog. 
+It contains a HyP3 plugin and an application for publishing jobs sent to it from the HyP3 plugin.
+
+## Usage
+
+To allow for a HyP3 job to publish its outputs, use the ingest-adapter container and add it as a step to the job spec file
+that looks like this:
+
+```yaml
+- name: PUBLISH
+  image: ghcr.io/ASFHyP3/ingest-adapter
+  command:
+    - --job-id
+    - Ref::bucket_prefix
```

Comment:
I assume the intention here is to pass `Ref::bucket_prefix` as the value for `--job-id`, but `job_id` is currently defined as a positional argument in [plugin.py](https://github.com/ASFHyP3/ingest-adapter/blob/develop/plugin/src/plugin.py), but I actually think it's more readable this way. So we should pick one of those two approaches.
```suggestion
    - --job-id
    - Ref::bucket_prefix
```